Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -218,6 +218,15 @@ protected def inhabited [Inhabited β] (e : α ≃ β) : Inhabited α := ⟨e.sy
 /-- If `α ≃ β` and `β` is a singleton type, then so is `α`. -/
 protected def unique [Unique β] (e : α ≃ β) : Unique α := e.symm.surjective.unique
 
+lemma Subtype.exists_congr {α β : Type*} {p : α → Prop} {q : β → Prop}
```

Comment:
I think `Equiv.exists_subtype_congr` would be a better name, to match `Equiv.exists_congr`. Can you mention that in the name, and perhaps put this lemma next to that one?